testsuite/cluster-test/src/experiments/state_sync_performance.rs

```diff
@@ -122,9 +122,9 @@ impl Experiment for StateSyncPerformance {
         // Calculate the state sync throughput
         let time_to_state_sync = start_instant.elapsed();
         let state_sync_throughput =
-            (validator_synced_version * 1000.0) / time_to_state_sync.as_millis() as f64;
+            validator_synced_version as f64 / time_to_state_sync.as_secs() as f64;
         let state_sync_throughput_message = format!(
-            "State sync throughput : {:?} txn/sec",
+            "State sync throughput : {:.2} txn/sec",
```

Comment:
I don't have the context in state-sync perf, but do we really need two decimal points here for transaction throughput (considering the experiments from cluster test fluctuate in the order 10 TPS)?

Reply:
Yeah, this decision was arbitrary. I'll round to the nearest integer, but indeed, there's so much variance in these runs that this level of precision doesn't matter.